app/Http/Controllers/PostController.php

```diff
@@ -19,25 +19,27 @@ public function getAllPosts()
 
     public function createPost(Request $request)
     {
-        $user = Auth::user();
-        $title = $request->title;
-        $body = $request->body;
-        $images = $request->images;
+        DB::transaction(function () use ($request) {
+            $user = Auth::user();
```

Comment:
@SlyCreator You're using the `DB` class here but you haven't imported it into this class file.

Reply:
This function would fail in this case as the DB class doesn't exist in this space, you'll need to use the DB facade in this file first.